tests/models/whisper/test_modeling_tf_whisper.py

```diff
@@ -650,9 +650,9 @@ def _test_large_logits_librispeech(in_queue, out_queue, timeout):
         input_speech = _load_datasamples(1)
 
         processor = WhisperProcessor.from_pretrained("openai/whisper-large")
-        processed_inputs = processor(audio=input_speech, text="This part of the speech", return_tensors="tf")
+        processed_inputs = processor(audio=input_speech, return_tensors="tf")
         input_features = processed_inputs.input_features
-        labels = processed_inputs.labels
+        labels = tf.convert_to_tensor([[5723, 644, 295, 220, 3322, 6218]])
```

Comment:
could you explain why we don't pass `text`, and use manual `labels`?

Reply:
Basically it’s just simpler to remove the prefix and suffix ☺️

Reply:
Got it.

If `removing prefix and suffix` is the necessary step to get the correct labels, when a user use the Whisper model, they has to do this manually, and cannot just 
```python
processed_inputs = processor(audio=input_speech, text="This part of the speech", return_tensors="tf")
labels = processed_inputs.labels
```
? I think the above code block should prepare the correct labels to be used for the models, otherwise we will get a lot of questions from the users in the future.

cc @sanchit-gandhi and @sgugger

Reply:
Well the above code is correct! But for testing, we used a special prompt, without the tokens that are necessary for the actual use of the model!🤗 
Overall, it's cleaner to just keep the logits and more resilient to changes in the tokenizer!

Reply:
Ah, it is used as `decoder_input_ids`!

Reply:
I'm in agreement with @ydshieh that we should use the text if possible (over hardcoding the labels).